src/Client.java

```diff
@@ -0,0 +1,34 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+
+public class Client {
+    public static void main(String args[]) {
+
+        try {
+            System.out.println(" Client has be started");
+            Socket soc = new Socket("localhost", 1024);
+            BufferedReader inputString = new BufferedReader(new InputStreamReader(System.in));
+            System.out.println("Enter your Message");
+            String str = inputString.readLine();
+            PrintWriter output = new PrintWriter(soc.getOutputStream(),true);
+            output.println(str);
+
+            BufferedReader in = new BufferedReader(new InputStreamReader(soc.getInputStream()));
+            System.out.println(in.readLine());
+            inputString.close();
```

Comment:
in also needs to be closed, and his will throw an exception if any variable is null. The close statements should go in a finally block, or you could use try-with-resources. https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html